docs/src/main/java/org/geotools/cql/ECQLExamples.java

```diff
@@ -145,6 +151,25 @@ private static void relatePattern() throws Exception{
 		
 	}
 
+    /**
+     * Example using the "contains" intersection matrix.
+     *
+     * @throws Exception
+     */
+    private static void referencedRelatePattern() throws Exception{
+
+        // ecql referencedRelatePattern start
+        Filter filter = ECQL.toFilter( "RELATE(geometry, SRID=4326;LINESTRING (-134.921387 58.687767, -135.303391 59.092838), T*****FF*)");
```

Comment:
Unwanted space between first "(" and '"".

Reply:
Yep